.circleci/config.yml

```diff
@@ -0,0 +1,44 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/rust:1.36.0
```

Comment:
Can we also do `stable`, `beta`, `nightly` here instead of a fixed version?

Reply:
Due to Circle CI docs [here](https://circleci.com/docs/2.0/circleci-images/#rust), it seems we don't have a choice to install `stable`, `beta`, `nightly` etc.

Reply:
I've changed the tag to `latest`, currently it might be the be choice to avoid a fixed version.